bin/zbackup

```diff
@@ -99,22 +99,44 @@ def snapshot(tier, filesystem, take_snapshot, keep, options):
     subprocess.check_call(zsnap_command)
 
 def replicate(filesystem, destination, options):
-    """Replicate given filesystem, possibly after deleting snapshots from other tiers."""
+    """Replicate given filesystem, possibly after deleting snapshots from other tiers.
```

Comment:
Locking ought to be optional, not mandatory.  Otherwise I foresee people posting bugs here saying "zbackup doesn't work anymore" and me having to give technical support.